src/modern/renderElement.js

```diff
@@ -0,0 +1,30 @@
+import invariant from 'invariant';
+import React from 'react';
+
+export default function renderElement(
+  match, Component, hasComponent, readyState,
+) {
+  const { route } = match;
+  const { props } = readyState;
+
+  if (!route.render) {
+    invariant(
+      hasComponent,
+      `Relay route ${route} specifies neither a render method nor a ` +
```

Comment:
This might make https://github.com/4Catalyzer/found/issues/105 unnecessary. The previous error message was just bad since it didn't say which route was busted. cc @rosskevin